src/iframe/IframeArticlePage.js

```diff
@@ -21,8 +21,26 @@ import { getAllDimensions } from '../util/trackingUtil';
 import PostResizeMessage from './PostResizeMessage';
 import FixDialogPosition from './FixDialogPosition';
 import { SocialMediaMetadata } from '../components/SocialMediaMetadata';
+import { fetchResource } from '../containers/Resources/resourceApi';
+import config from '../config';
 
 class IframeArticlePage extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      contentUrl: '',
+    };
+  }
+
+  componentDidMount() {
+    const resourceId = this.props.location.pathname.split('/')[3];
+    fetchResource(resourceId).then(resource => {
```

Comment:
Er det ikkje mogeleg for emner å være embedda på denne måten?

Reply:
Hva mener du her?

Reply:
Ah. Det finnes en paralell IframeTopicPage som viser emneartikler. Denne må også oppdateres med logo og lenke.

Reply:
fetchResource henter ressurs fra taksonomi. IframeTopicPage må hente topic fra taksonomi på samme måte.